src/App.jsx

```diff
@@ -0,0 +1,16 @@
+import TodoItems from "./components/TodoItems";
+import store from "./redux/store/configureStore.js";
+
+import { Provider } from "react-redux";
+
+function App() {
+  return (
+    <>
```

Comment:
remove these unnecessary fragments.
check-in all components, use only where you need.